.circleci/config.yml

```diff
@@ -126,18 +126,52 @@ jobs:
       - store_artifacts:
           path: ~/artifacts
           destination: packages
-      - deploy:
-          name: Maybe Deploy
+      - persist_to_workspace:
+          root: /home/circleci/artifacts
+          paths:
+            - trusty
+            - xenial
+            - el6
+            - el7
```

Comment:
Is it possible to simplify but just using `.` instead of hardcoding every path?
I guess there is nothing else in artifacts dir and so additional filtering is not needed.

Reply:
Looks like artifacts dir also contains logs:

<img width="412" alt="image" src="https://user-images.githubusercontent.com/3607046/46299854-23e77c80-c557-11e8-9eca-bb55fb69634c.png">

Probably not all that big a deal if we also persist those to workspace. Would need to check what st2flow and bwc-ui repos are doing here too, since they follow the same pattern.

Reply:
Will leave this as current pattern for now, if we change it to be less specific, we will do it as change across all repos where we use this design pattern